### ISSUE TITLE
Tikz overlay breaks footer

### ISSUE THREAD

**Issue by josephwright** [josephwright/beamer#278]
Hello, the following example produces a wrong result:
```
\documentclass{beamer}

\usepackage{tikz}

\usetheme{Madrid}

\begin{document}

\begin{frame}% [fragile] -- doesn't help

  Frame 1
  \pause

  Frame 2
  \begin{tikzpicture}[overlay]
    \onslide<1->
  \end{tikzpicture}
  \pause

  Frame 3
\end{frame}

\end{document}
```
In the first frame, bevor the overlay is shown, the footer is missing. The other 2 slides render correctly. The PDF and images showing my result can be found here I tried 3 PDF viewers, so i really do think this is a problem within beamer. I attached the compilation log. The problem occurs with the most recent texlive version and with the current bitbucket head.

----------------------------------------
- Bitbucket: https://bitbucket.org/rivanvx/beamer/issue/278
- Originally reported by: thomas001le 
- Originally created at:  2013-12-10

**Comment by josephwright** [comment]
_From thomas001le on 2012-12-10_



    Note: The `\onslide<1->` was put in there as some stackoverflow posts suggested this could solve the issue, which does not work here. the error does also occur without the `\onslide<1->`

    Note: Tracked it down to 3eb0f325483edb55a314e40b24215769940f9ee9. this commit introduces the problem.

**Comment by josephwright** [comment]
_From Joseph Wright on 2013-12-16_

I see no difference before and after the commit you cite: could you send me PDFs showing the effect (or attach them)?

**Comment by josephwright** [comment]
_From Joseph Wright on 2013-12-17_

Ah, I see what you are getting at: if you remove the `\onslide<1->` business then there is a difference in behaviour. I'll have to look at how TikZ implements overlay, I guess.

**Comment by josephwright** [comment]
_From Joseph Wright on 2013-12-17_

No, not an overlay issue: something else: investigating

**Comment by josephwright** [comment]
_From thomas001le  on 2013-12-17_

Build with 3eb0f325483edb55a314e40b24215769940f9ee9, not footer on first slide. I get this result whether i include \onslide<1-> or not. Also i get the same result with the most recent revision of beamer.

**Comment by josephwright** [comment]
Fixed by 62f63b53f3d9f5d01448ace59e35d2614f4d89dd.
